packages/compiler-cli/ngcc/src/entry_point_finder/targeted_entry_point_finder.ts

```diff
@@ -147,6 +147,18 @@ export class TargetedEntryPointFinder extends TracingEntryPointFinder {
     return this.computePackagePathFromNearestNodeModules(entryPointPath);
   }
 
+  /**
+   * Compute whether the `test` path is contained within the `base` path.
+   *
+   * Note that this doesn't use a simple `startsWith()` since that would result in a false positive
+   * for `test` paths `a/b/c-x` when the `base` path is `a/b/c`.
```

Comment:
Should this be like this? It reads a bit strange currently.

```suggestion
   * for `test` path `a/b/c-x` when the `base` path is `a/b/c`.
```

Reply:
Actually I meant it to be:

```suggestion
   * for `test` paths such as `a/b/c-x` when the `base` path is `a/b/c`.
```